eng/targets/GenerateCompilerExecutableBindingRedirects.targets

```diff
@@ -14,7 +14,7 @@
           DependsOnTargets="ResolveAssemblyReferences"
           Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'">
     <ItemGroup>
-      <SuggestedBindingRedirects Include="@(ReferencePath->'%(FusionName)')" MaxVersion="%(ReferencePath.Version)" KeepMetadata="-none-" Condition="'%(ReferencePath.FrameworkFile)' != 'true'" />
+      <SuggestedBindingRedirects Include="@(ReferencePath->'%(FusionName)')" MaxVersion="%(ReferencePath.Version)" KeepMetadata="-none-" Condition="'%(ReferencePath.FrameworkFile)' != 'true' AND '%(ReferencePath.Version)' != ''" />
```

Comment:
Nit: lower case `AND`

Reply:
I'd also add a comment saying that the Version is not available in design time builds, just so if anyone wonders why the condition is in place.

Reply:
Done thanks!